Source/CBL_Pusher.m

```diff
@@ -279,6 +279,12 @@ - (void) processInbox: (CBL_RevisionList*)changes {
                             [self revisionFailed];
                             continue;
                         }
+                        
+                        if ([loadedRev[@"_removed"] boolValue]) {
```

Comment:
This will throw an exception if the `_removed` property contains anything but a NSNumber. Probably best to compare it to `@YES`.